spec/controllers/idv/verify_info_controller_spec.rb

```diff
@@ -393,21 +453,40 @@
 
       context 'when threatmetrix response is Reject' do
         let(:review_status) { 'reject' }
+        let(:success) { false }
 
         it 'sets the review status in the idv session' do
           get :show
           expect(controller.idv_session.threatmetrix_review_status).to eq('reject')
         end
 
         it 'tracks a failed tmx fraud check' do
-          stub_attempts_tracker
           expect(@attempts_api_tracker).to receive(:idv_tmx_fraud_check).with(
-            success: false,
+            success:,
             failure_reason: {
               tmx_summary_reason_code: ['Identity_Negative_History'],
             },
           )
 
+          expect(@attempts_api_tracker).to receive(:idv_verification_submitted).with(
+            success: true,
```

Comment:
should this be passing? since it fails threatmetrix with a "rejected" value?

Reply:
i think it should be false here

Reply:
based on this [research i did yesterday](https://gsa-tts.slack.com/archives/C05HSH9RQ57/p1747334937110059?thread_ts=1747329733.461799&cid=C05HSH9RQ57), it looks like `true` is correct -- we handle the results of a failed fraud review downstream. it's confusing, so i added a comment in the `ResultAdjucator` class for anyone confused in the future.